### ISSUE TITLE
Media as Images

### ISSUE THREAD

**Issue by trilodge** [SAP/fundamental-styles#414]
**Is your feature request related to a problem? Please describe.**

Images are rendered as background-images with an inline style="" attribute into the mentioned components. 

> Is this issue related to a specific component?

Image and Tile components

> What browser are you working in?

Firefox, Chrome & Edge

> What version of Fundamental Styles are you using?

Latest

**Describe the solution you'd like**

Images should be rendered as `<img>` or better as `<figure>` + `<figcaption>` for the Tile component, which would improve the semantic markup for the component if images are included. For the Image component images should be rendered as `<picture>` or as `<img>` with srcset attributes. 
To make the images not destroy the layout css `object-fit` could be used to get the same results as `background-size: cover;` like it is rendered now with the background-image approach. 

Also accessibility would massivly improve for images. Also rendering performance would benefit. 

Other things which would benefit from this: 

- `<img>` or any other proposed solution won't trigger a complete css repaint and layout trashing.
- `<img>` or any other proposed solution could provide an `alt=""` attribute and this would be used by screen readers or any other assitive technology, also those alt attributes are rendered if the image doesn't load.
- `<img>` or any other proposed solution could be rendered resolution and display density specific with only providing the assets which fit best for the user. With rendering `srcset=""` or `<source>` we could offer better images for screen sizes or retina displays. 
- HTML5 introduced `<figure>` and `<figcaption>` tags to render images like used in the tile component.

**Additional context**
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure
https://developer.mozilla.org/en/docs/Web/CSS/object-fit
https://developer.mozilla.org/en/docs/Web/HTML/Element/picture
https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

Note:
Since IE11 doesn't support srcset or picture but everytime the fallback with img src="" this browser would need some extra efforts. Also applies to object-fit.

**Comment by droshev** [comment]
this issue was already addressed
